uiowa/uiowa
# Scholarship Feature & Admissions Site Split/Customizations

Resolves #2214

- Establishes Scholarship content type feature split

## Custom scholarship additions for Admissions via site split:

- A single scholarships block view that takes multiple contextual filters on scholarship type/resident.
- Custom block to pull together the scholarships view block with easy UX for providing arguments. 
- Custom block "jump menu" leveraging the Forms API instead of the odd webform/javascript solution that currently exists. Allows configurable destination paths based on entity reference.
- Both blocks can be place where ever allowing content around it.
- Scholarship teaser displays all data per requirements in an accordion listing.

## To Test on v3 sites

- `blt dsa`
- `drush @alias.local config:import --source ../config/features/scholarship --partial`
- Enable the scholarships split admin/config/development/configuration/config-split
- As a webmaster, you are able to create scholarships. All the fields except type/resident are available. The permissions assume that this is a single extension of the v3 default. Adjustments might be needed for v2, collegiate or other combinations of splits.

## To Test On Admissions

- `blt ds --site=admissions.uiowa.edu`
- `drush @admissions.local config:import --source ../config/features/scholarship --partial`
- Enable the scholarships split admin/config/development/configuration/config-split
- `drush @admissions.local config:import --source ../config/admissions.uiowa.edu --partial`
- `drush @admissions.local cim`
- As a webmaster, you are able to create scholarships. All the fields from the issue are accounted for.
- As a webmaster, you are able to see Admissions Core blocks in the More.. section of layout builder and you are able to place and configure. The jump menu block takes you to the paths you have specified. The scholarships block displays scholarships in a working accordion that meets the filter criteria.
